apps/studio/components/interfaces/Integrations/Queues/QueuesListing.tsx

```diff
@@ -41,7 +41,7 @@ export const QueuesListing = () => {
 
   const pgmqExtension = (extensions ?? []).find((ext) => ext.name === 'pgmq')
   // TODO: Change this to true for local development to work
-  const pgmqExtensionInstalled = pgmqExtension?.installed_version
+  const pgmqExtensionInstalled = true // pgmqExtension?.installed_version
```

Comment:
Just leaving this here so we don't forget before merging!
```suggestion
  const pgmqExtensionInstalled = Boolean(pgmqExtension?.installed_version)
```

Reply:
thank youuu